code/__HELPERS/_macros.dm

```diff
@@ -5,8 +5,30 @@
 // fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
 #define ishuman(A) istype(A, /mob/living/carbon/human)
 
+#define isjusthuman(A) A.type == /mob/living/carbon/human
+
 #define ismonkey(A) istype(A, /mob/living/carbon/monkey)
 
+#define isvox(A) istype(A, /mob/living/carbon/human/vox)
```

Comment:
@Shadowmech88 technically macros can be any amount of lines long.

They're just text replacers however, so in the use case of `isX()`, you'll be indeed limited to anything you can put in an expression.